### ISSUE TITLE
CLI: Errors occurred processing action add

### ISSUE THREAD

**Issue by mrturcot** [vaeth/mv-overlay#29]
Hello, I cannot add mv overlay. Getting this below. Thanks!

```
layman -a mv

 * Adding overlay...
 * Overlay "mv" is not official. Continue installing? [y/n]: 
 * Please respond with [y/n]: y
 * Running Git... # ( cd /var/lib/layman  && /usr/bin/git clone https://anongit.gentoo.org/git/user/mv.git /var/lib/layman/mv )
Cloning into '/var/lib/layman/mv'...
fatal: unable to access 'https://anongit.gentoo.org/git/user/mv.git/': Could not resolve host: anongit.gentoo.org
 * Failure result returned from Git
 * Running Git... # ( cd /var/lib/layman/mv  && /usr/bin/git config user.name "layman" )
 * [Errno 2] No such file or directory: '/var/lib/layman/mv'
 * 
 * Trying next source of listed sources...
 * Running Git... # ( cd /var/lib/layman  && /usr/bin/git clone git://anongit.gentoo.org/user/mv.git /var/lib/layman/mv )
Cloning into '/var/lib/layman/mv'...
fatal: unable to look up anongit.gentoo.org (port 9418) (Temporary failure in name resolution)
 * Failure result returned from Git
 * Running Git... # ( cd /var/lib/layman/mv  && /usr/bin/git config user.name "layman" )
 * [Errno 2] No such file or directory: '/var/lib/layman/mv'
 * 
 * Trying next source of listed sources...
 * Running Git... # ( cd /var/lib/layman  && /usr/bin/git clone git+ssh://git@git.gentoo.org/user/mv.git /var/lib/layman/mv )
Cloning into '/var/lib/layman/mv'...
git@git.gentoo.org: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
 * Failure result returned from Git
 * Running Git... # ( cd /var/lib/layman/mv  && /usr/bin/git config user.name "layman" )
 * [Errno 2] No such file or directory: '/var/lib/layman/mv'
 * Adding repository "mv" failed!

 * CLI: Errors occurred processing action add
 * Adding repository "mv" failed!
```

**Comment by vaeth** [comment]
Apparently anongit.gentoo.org has currently a problem with being accessed. Perhaps it was renamed permanently. This happens all by gentoo, I have no influence on this.

You can use the github repository as well; I keep them always in sync.

**Comment by mrturcot** [comment]
> You can use the github repository as well; I keep them always in sync.

Thank you, I actually never new that was an option 😅 However this `layman -o https://raw.githubusercontent.com/vaeth/mv-overlay/main/repositories.xml -f -a mv` worked like a charm. Cheers!

**Comment by mrturcot** [comment]
Maybe I spoke too soon? I ran the above I got this

```
[5] "" /var/lib/layman/mv (cache: parse|ebuild*3.0.24#metadata-md5#metadata-flat#assign)
     Reading category 177|177 (100) EMPTY!
```

And

```
 * Running Git... # ( cd /var/lib/layman/mv  && /usr/bin/git pull )
There is no tracking information for the current branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>

If you wish to set tracking information for this branch you can do so with:

    git branch --set-upstream-to=origin/<branch> master

 * Failure result returned from Git
```

Am I doing something wrong?

**Comment by vaeth** [comment]
I think there's a bug in layman that it assumes that the main branch is called “master”.
The mv overlay follows the new default name in git which is "main” instead, see e.g. [this article](https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main).

**Comment by mrturcot** [comment]
> I think there's a bug in layman that it assumes that the main branch is called “master”.
> The mv overlay follows the new default name in git which is "main” instead, see e.g. [this article](https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main).

Roger that captain. Thank you :)
